development/webpack/utils/cli.ts

```diff
@@ -394,13 +409,19 @@ function getOptions(
 /**
  * Returns a string representation of the given arguments and features.
  *
- * @param args
- * @param features
+ * @param args - The parsed CLI arguments
+ * @param features - The active and available features
+ * @param resolvedEnvironment - The resolved MetaMask environment
  */
-export function getDryRunMessage(args: Args, features: Features) {
+export function getDryRunMessage(
+  args: Args,
+  features: Features,
+  resolvedEnvironment?: string,
```

Comment:
Was this meant to be optional? It appears to be passed in 100% of the time.

```suggestion
  resolvedEnvironment: string,
```